lib/post-process/src/lib.rs

```diff
@@ -41,6 +41,24 @@ fn enforce_prefix<'a>(lexer: &Lexer<'a, PostElement<'a>>) -> bool {
     }
 }
 
+#[inline]
+fn emoji_split<'a>(lexer: &Lexer<'a, PostElement<'a>>) -> Option<(&'a str, Option<&'a str>)> {
+    if !enforce_prefix(lexer) || !enforce_postfix(lexer) {
```

Comment:
Actually we don't need to enforce prefixes and postfixes here, since custom emojis are valid, even when not separated properly from other post components